stable/keycloak/scripts/datasource.cli

```diff
@@ -0,0 +1,12 @@
+## Query timeout in seconds
+/subsystem=datasources/data-source=KeycloakDS/:write-attribute(name=query-timeout,value=${env.DB_QUERY_TIMEOUT:300})
+
+## Postgres specific connection validation
```

Comment:
Postgres-specific stuff should only be added when Postgres is selected.

Reply:
Added guard for postgres specific config.